traitsui/tests/editors/test_text_editor.py

```diff
@@ -317,10 +322,10 @@ def check_format_func_used(self, style):
         tester = UITester()
         with tester.create_ui(foo, dict(view=view)) as ui:
             display_name = (
-                tester.find_by_name(ui, "name").inspect(query.DisplayedText())
+                tester.find_by_name(ui, "name").inspect(DisplayedText())
             )
             display_nickname = (
-                tester.find_by_name(ui, "nickname").inspect(query.DisplayedText())  # noqa E501
+                tester.find_by_name(ui, "nickname").inspect(DisplayedText())  # noqa E501
```

Comment:
Nit: Looks like the `noqa E501` can be removed from here now. (I have not verified.)